### PR TITLE
New Tracker based on Hungarian algorithm

### PR DESCRIPTION
**Pull request by Twarz**
- The old tracker was not optimized for many people
- The id is now assigned with uuid each time a new person is detected (instead of integer) 
- The tracker is generic and can be used for other purposes

### CODE REVIEW

File: rt_gene/launch/estimate_gaze.launch [hunk 1]
```diff
@@ -24,7 +24,6 @@
             args="0.0 0.0 0.0 0.0 0.0 0.0 1.0 $(arg rgb_frame_id_ros) $(arg rgb_frame_id)" unless="$(arg is_frame_rotated)" />
 
       <node pkg="rt_gene" type="extract_landmarks_node.py" name="extract_landmarks_new" output="screen">
-          <param name="gpu_id"  value="$(arg gpu_id)" />
```
[the remark this answers is not in the document]
Comment (Tobias-Fischer) [reply rt_gene/launch/estimate_gaze.launch]: See https://github.com/Tobias-Fischer/rt_gene/commit/90afa6546684933c671e8edcede6c4e9c49f7594

**Comment by Twarz** [comment]
Hey @Tobias-Fischer,
I rebased the last commits over your last one and resolved small conflicts. I also made the changes you mentioned.
You should be able to merge it properly now :)

Best,
Kévin

**Comment by Tobias-Fischer** [comment]
Looks all good to me, merged, thanks a lot!

**Comment by ngageorange** [comment]
Sorry to being late to this pull request. This breaks downstream code as the subject_ids are now uuids rather than being the index in an array.

Could we reconcile this new algorithm to provide compatibility with downstream code?

**Pull request merged**